README.md

```diff
@@ -1,312 +1,312 @@
-| Ch3b |     |                                |
-| Ch4a |     |                                |
-| Ch4b |     |                                |
-| Ch5a |     |                                |
-| Ch5b |     |                                |
-| Ch6a |     |                                |
-| Ch6b |     |                                |
-| Ch7a |     |                                |
-| Ch7b |     |                                |
-
-## Slot 6 - [NI 9871](https://www.ni.com/en-us/support/model.ni-9871.html)
-
-4 port RS-485/422. Connected devices are usually using Modbus connection.
-
-| Port | Pin | Assignment                     |
-| ---- | --- | ------------------------------ |
-| 1    | NA  | Coolant Pump VFD               |
-| 2    | NA  | Flow meter                     |
-| 3    | NA  | Wind sensor                    |
-| 4    | NA  |                                |
-
-## Slot 7 - [NI 9870](https://www.ni.com/en-us/support/model.ni-9870.html)
-
-| Port | Pin | Assignment                     |
-| ---- | --- | ------------------------------ |
-| 1    | NA  |                                |
-| 2    | NA  |                                |
-| 3    | NA  |                                |
-| 4    | NA  |                                |
-
-## Slot 8 - [NI 9213](https://www.ni.com/en-us/support/model.ni-9213.html)
-
-| port | Pins(+-) | Assignment               |
-| ---- | -------- | ------------------------ |
-| TC0  | 2,20     | air mirror top surface   |
-| TC1  | 3,21     | mirror cell air          |
-| TC2  | 4,22     | mirror coolant supply    |
-| TC3  | 5,23     | mirror coolant return    |
-| TC4  | 6,24     | telescope coolant supply |
-| TC5  | 7,25     | telescope coolant return |
-| TC6  | 8,26     |                          |
-| TC7  | 9,27     |                          |
-| TC8  | 10,28    |                          |
-| TC9  | 11,29    |                          |
-| TC10 | 12,30    |                          |
-| TC11 | 13,31    |                          |
-| TC12 | 14,32    |                          |
-| TC13 | 15,33    |                          |
-| TC14 | 16,34    |                          |
+# ts_M1M3ThermalFPGA
+
+## Description
+
+This repo contains the LabVIEW 2018 code of the M1M3 Thermal System FPGA used
+by the **ts_M1M3Thermal** software. The FPGA is middleman between [C/C++
+control application](https://github.com/lsst-ts/ts_m1m3thermal) and hardware
+connected to cRIO.
+
+```mermaid
+graph LR
+Controller -- CommandFIFO --> FPGA
+FPGA -- CommandFIFOMultiplexer --> DIO
+Controller -- RequestFIFO --> FPGA
+FPGA -- U8ResponseFIFO --> Controller
+FPGA -- U16ResponseFIFO --> Controller
+```
+
+:warning: Please use **git clone --recursive** to get linked dependencies (Common_ libraries).
+
+## LabView Dependencies
+
+* FPGA Support
+* cRIO support
+* Real Time support
+* [NI Tools Network](https://www.ni.com/labview-tools-network)
+  * LabView FPGA Floating-Point Library by NI

```

Comment:
Is the "space" intentional here?

Reply:
Yes. Floating library is installed through NI Tools Network.